### ISSUE TITLE
[RFE] sss_cache should be able to clear ID-mappings

### ISSUE THREAD

**Issue by pbrezina** [pbrezina/sssd-test#1891]
Cloned from Pagure issue: https://pagure.io/SSSD/sssd/issue/1887
- Created at 2013-04-19 14:43:03 by [sgallagh](https://pagure.io/user/sgallagh)
- Closed at 2020-03-24 14:11:58 as wontfix
- Assigned to nobody
---

We're planning in ticket #1568 to allow storing the objectSID <-> POSIX ID mapping in the sysdb in order to be able to perform tokenGroups lookups even when using assigned POSIX IDs. We should also be able to modify the cache to remove this mapping (and reassign it on the next lookup) so we can address the rare case when a POSIX ID changes in AD.

### Comments

---

**Comment from [sgallagh](https://pagure.io/user/sgallagh) at [2013-04-25 15:47:12](https://pagure.io/SSSD/sssd/issue/1887#223278)**

Fields changed

milestone: NEEDS_TRIAGE => SSSD 1.12 beta
rhbz:  => todo

---

**Comment from [sgallagh](https://pagure.io/user/sgallagh) at [2013-12-02 21:23:11](https://pagure.io/SSSD/sssd/issue/1887#223279)**

Fields changed

changelog:  => 
summary: sss_cache should be able to clear ID-mappings => [RFE] sss_cache should be able to clear ID-mappings

---

**Comment from [sgallagh](https://pagure.io/user/sgallagh) at [2016-02-16 13:35:05](https://pagure.io/SSSD/sssd/issue/1887#223280)**

In 1.14 we will provide a way to clear the cache with sss_cache, but not the mappings per se.

mark:  => 0
milestone: SSSD 1.14 beta => SSSD 1.15 beta
sensitive:  => 0

---

**Comment from [sgallagh](https://pagure.io/user/sgallagh) at [2017-02-24 14:27:43](https://pagure.io/SSSD/sssd/issue/1887#223281)**

**Metadata Update from @sgallagh**:
- Issue marked as depending on: #1568
- Issue set to the milestone: SSSD Future releases (no date set yet)

---

**Comment from [sgallagh](https://pagure.io/user/sgallagh) at [2020-03-13 14:03:41](https://pagure.io/SSSD/sssd/issue/1887#633169)**

**Metadata Update from @thalman**:
- Custom field blocking adjusted to 1568
- Custom field design_review reset (from 0)
- Custom field mark reset (from 0)
- Custom field patch reset (from 0)
- Custom field review reset (from 0)
- Custom field sensitive reset (from 0)
- Custom field testsupdated reset (from 0)
- Issue close_status updated to: None
- Issue tagged with: Canditate to close

---

**Comment from [sgallagh](https://pagure.io/user/sgallagh) at [2020-03-24 14:11:57](https://pagure.io/SSSD/sssd/issue/1887#636668)**

Thank you for taking time to submit this request for SSSD. Unfortunately this issue was not given priority and the team lacks the capacity to work on it at this time.

Given that we are unable to fulfill this request I am closing the issue as wontfix.

If the issue still persist on recent SSSD you can request re-consideration of this decision by reopening this issue. Please provide additional technical details about its importance to you.

Thank you for understanding.

---

**Comment from [sgallagh](https://pagure.io/user/sgallagh) at [2020-03-24 14:11:59](https://pagure.io/SSSD/sssd/issue/1887#636669)**

**Metadata Update from @pbrezina**:
- Issue close_status updated to: wontfix
- Issue status updated to: Closed (was: Open)
